src/Cassandra/Orleans.Clustering.Cassandra/OrleansQueries.cs

```diff
@@ -42,6 +44,9 @@ private OrleansQueries(ISession session)
 
     public ConsistencyLevel MembershipReadConsistencyLevel { get; set; }
 
+    // TODO Any subsequent updates to the column resets the TTL
```

Comment:
The update/insert should probably have a version that sets TTL, too, to address this

Reply:
I think if I also set the table default TTL (to the same value), that should account for any updates or inserts, to apply the default TTL to any cell that is inserted/changed?

And taking that further, that probably means the IAmAlive doesn't need to do `USING TTL` either because the table's default TTL would be applied anyway.